### DISCUSSION TITLE
Teszt rendszeren arrivalNumber null

### DISCUSSION THREAD

**Discussion by czeba-b** [nav-gov-hu/M2M#43] [Kérdések] [answered]
Sziasztok!

A teszt rendszeren próbálok T1042E nyomtatványt beküldeni. Úgy látom hogy a validációk rendben vannak, mégsem kapok érkeztetési számot.
createDocument eredménye:
```json
{
   "resultMessage":null,
   "documentStatus":"VALIDATED",
   "errors":null,
   "resultCode":"CREATE_DOCUMENT_SUCCESS"
}
```
getDocument eredménye:
```json
{
   "resultMessage":null,
   "documentStatus":"VALIDATED",
   "documentFileId":"2a703465-bf4c-4b35-93a9-51290515fd4e",
   "errors":null,
   "arrivalNumber":null,
   "resultCode":"GET_DOCUMENT_SUCCESS"
}
```
Próbáltam pár percet várni és újra lekérdezni a getDocument-el az állapotot, de ugyanezt kapom.

Elrontok valamit vagy ez még nem a folyamat vége és kell hívni egy másik API-t a beküldéshez?

**Comment by TamasPill** [comment, marked as answer]
Igen, még az updateDocument-et meg kell hívni az UNDER_SUBMIT státusszal.

**Reply by czeba-b** [reply]
Köszönöm a gyors választ, ez hiányzott.
